### DISCUSSION TITLE
如何在mmcls的基础上构造一维卷积神经网络？

### DISCUSSION THREAD

**Discussion by xingzhe996** [open-mmlab/mmclassification#1088] [Q&A] [answered]
### 我已试图查询该问题

- [X] 我已经查阅了相关 issue，没有找到相关帮助。
- [X] 我已经查询了[官方文档](https://mmclassification.readthedocs.io/zh_CN/master/)，没有找到相关帮助。

### 描述该问题

我再、准备修改mmcls中的EfficientNet的时候，发现构造的卷积使用的是from mmcv.cnn.bricks import ConvModule，但是ConvModule无法控制是1维卷积还是2维卷积，请问应该如何处理呢？

**Comment by mm-assistant** [comment]
We recommend using English or English & Chinese for issues so that we could have broader discussion.

**Comment by Ezra-Yu** [comment]
是只输入的图片不是rgb 3通道图片，只有一个通道，类似灰度图这种吗？

**Comment by xingzhe996** [comment]
不是的，我是想把efficient网络改成可以输入一个序列的，就是一个一维向量，然后网络里面应该用的是conv1d，但是我没找到mmcls里面构造网络时用的conv2d在哪里？mmcls里面构造时用的是from mmcv.cnn.bricks import ConvModule，但是一个卷积模型无法设置为1d卷积

**Comment by Ezra-Yu** [comment, marked as answer]
try:
```
ConvModule(....., conv_cfg=dict(type='Conv1d')   ,......)
```

**Comment by xingzhe996** [comment]
好的，非常感谢帮忙
